generate-test-vector.py

```diff
@@ -1241,6 +1241,51 @@ def generate_taproot_with_nums_point_test():
 
     return test_cases
 
+def generate_pay_to_multisig_test():
+    sender, recipient, test_case = new_test_case()
+    outpoint = ("f4184fc596403b9d638783cf57adfe4c75c605f6356fbc91338530e9831e9e16", 0)
+    sender_bip32_seed = 'deadbeef'
+    recipient_bip32_seed = 'f00dbabe'
+    scan, spend, Scan, Spend = reference.derive_silent_payment_key_pair(bytes.fromhex(recipient_bip32_seed))
+    address = reference.encode_silent_payment_address(Scan, Spend, hrp=HRP)
+    addresses = [(address, 1.0)]
+
+    sender['given']['recipients'] = addresses
+    recipient['given']['key_material']['scan_priv_key'] = scan.get_bytes().hex()
+    recipient['given']['key_material']['spend_priv_key'] = spend.get_bytes().hex()
+    recipient['expected']['addresses'] = [address]
+
+    keys = [
+        get_key_pair(0, seed=bytes.fromhex(sender_bip32_seed)),
+        get_key_pair(1, seed=bytes.fromhex(sender_bip32_seed))
+    ]
+    # OP_2 pub1 pub2 OP_3 OP_EQUAL
+    multisig_script = "52" + ''.join(["21"+key[1].get_bytes(False).hex() for key in keys]) + "53ae"
+    p2sh = f"a914{hash160(bytes.fromhex(multisig_script)).hex()}87"
+
+    msg = hashlib.sha256(b'message').digest()
+    sigs = ["47" + key[0].sign_ecdsa(msg, low_s=False, rfc6979=True).hex() + "01" for key in keys]
```

Comment:
not sure if the signatures are always 71 bytes (`0x47`), might want to verify. if they aren't, @S3RK has some code in the `generate_all_inputs` test for automatically picking the correct push opcode based on the size of the signature.